chore: Edit Displayed Text in Bug List Filters Fix

harsha-mangena opened the pull request:
- issue https://github.com/Niloth-p/glitch-guru/issues/1
- used constants for the text displayed in filters

harsha-mangena committed:
- issue https://github.com/Niloth-p/glitch-guru/issues/1
- used constants for the text displayed in filters

Niloth-p reviewed (changes requested):
Those (the choices) are not dictionaries.
I recommend you to setup the project locally and run the server with your changes so that you can debug syntax errors like this on your own.

harsha-mangena committed:
- added a util function to get actual value from list of tuples.

harsha-mangena commented:
@Niloth-p 
I kind of assumed assumed those were dictionaries.
Added util function for getting the actual text from the list.

harsha-mangena commented:
@Niloth-p  Can you kindly review this, And please let me know If any changes are required.

Niloth-p committed:
Update utils.py

- Simplify the function get_choice_display()
- Improve function docstring
- Add module docstring

Niloth-p committed:
Merge branch 'main' into edit-text-displayed-in-filters-fix-harsha
